templates/github/.github/workflows/scripts/install.sh.j2

```diff
@@ -141,6 +141,8 @@ if [[ " ${SCENARIOS[*]} " =~ " ${TEST} " ]]; then
   - name: pulp-fixtures\
     image: docker.io/pulp/pulp-fixtures:latest\
     env: {BASE_URL: "http://pulp-fixtures:8080"}' vars/main.yaml
+
+  export REMOTE_FIXTURES_ORIGIN="http://pulp-fixtures:8080"
```

Comment:
So this is only in case `docker-fixtures` is set. So the `https://fixtures.pulpproject.org/` is the default value in the  corresponding pulpcore fixture, right?

Reply:
Yes. Default value is `https://fixtures.pulpproject.org/`.